libs/chain-signatures/contract/src/state/running.rs

```diff
@@ -144,7 +144,7 @@ impl RunningContractState {
         }
         let participant = AuthenticatedParticipantId::new(self.parameters.participants())?;
         let n_votes = self.add_domains_votes.vote(domains.clone(), &participant);
-        if self.parameters.threshold().value() <= n_votes {
+        if self.parameters.participants().len() as u64 == n_votes {
```

Comment:
Could you bind verbose or complex expressions to a descriptive variable name so it's easier to follow as a reader?

For example:
```suggestion
        let threshold_number_of_votes_casted = self.parameters.participants().len() as u64 == n_votes;
        if threshold_number_of_votes_casted {
```

Reply:
In this case I would argue that extracting it out is harder to read. I think the code as it is is very clear already. It spells "the number of participants equals the number of votes".

Reply:
My confusion comes from the naming of `n_votes`. `n_votes` is not descriptive enough to tell me what it means, so I don't understand what the if check does.

Now that you tell me it means number of casted votes, it's also obvious to me what the if check does.

Naming our variables something more descriptive from the get go, for example `number_of_casted_votes`, then it would be easy to understand without needing a comment/explanation or a variable.

```rust
 if self.parameters.participants().len() as u64 == casted_votes {
```

Reply:
> My confusion comes from the naming of n_votes. n_votes is not descriptive enough to tell me what it means, so I don't understand what the if check does.

The `n` is often used as shorthand for `num` or `number`.

I think we’re landing in the realm of personal preference here. I usually lean toward shorter variable names like `n_votes` to keep the code compact and easy to scan. While `number_of_casted_votes` is more explicit, it also gives the impression of precision that isn’t entirely accurate—we're not counting all casted votes, but rather the votes for this specific proposal. So to be fully accurate, it would have to be something like `number_of_casted_votes_for_proposal`, which feels overly verbose to me.